src/sentry/integrations/slack/actions/notification.py

```diff
@@ -70,6 +70,8 @@ def send_notification(event: GroupEvent, futures: Sequence[RuleFuture]) -> None:
                         "text": blocks.get("text"),
                         "blocks": json.dumps(blocks.get("blocks")),
                         "channel": channel,
+                        "unfurl_links": False,
+                        "unfurl_media": False,
```

Comment:
addresses `slack.event.unfurl-error (error='cannot_parse_attachment')` that i came across when trigger issue alerts on my local configuration